src/lib.rs

```diff
@@ -487,7 +487,7 @@ impl<TypeState> ConnectionOptions<TypeState> {
     /// to apply the desired configuration to all server connections.
     ///
     /// # Examples
-    /// ```
+    /// ```no_run
```

Comment:
Why no run?

Reply:
sometimes the test fails from my home connection to the demo server. I'll run an mtr next time and find out why.

Reply:
We must have a timeout of some sort too low. Since we are going Germany to west coast US. Let's fix that and keep running it imo.

Reply:
Loss sometimes generally just gets kind of high depending on the route. Today's weather event seems to be around the Telia backbone. Often there are weather events related to poor network maintainership of German ISPs. So, in this case it seems to be more of a normal internet weather thing. It errors out fairly frequently and tends to be a source of false-negative stress.
![image](https://user-images.githubusercontent.com/1505488/80385115-f800ba00-88a5-11ea-9854-d07e82f98e2b.png)

Reply:
How long does the client try to connect currently?